src/components/VNepaliDatePicker.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <div class="wrapper">
-        <input type="text" placeholder="yyyy-mm-dd" ref="elm" :value="dateValue" :class="inputClasses"/>
+        <input type="text" placeholder="yyyy-mm-dd" ref="elm" :value="dateValue" :class="inputClasses" :disabled="disabled"/>
         <slot v-if="allowClear" :on-clear="onClear" name="clear-btn">
```

Comment:
@Sandip124 I think we need to handle the "clear-btn" slot as well. If the "disabled" props is true, maybe we should not render the slot as well.